src/components/content/systemStatus/DisplayHealthCheckStatus.tsx

```diff
@@ -0,0 +1,217 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Alert, Button, Space, Table, TableProps } from 'antd';
+import { ColumnsType } from 'antd/es/table';
+import { AdminService, ApiError, BackendSystemStatus, Response, SystemStatus } from '../../../xpanse-api/generated';
+import { DashboardOutlined } from '@ant-design/icons';
+import SystemStatusIcon from './SystemStatusIcon';
+import { ColumnFilterItem } from 'antd/es/table/interface';
+import { convertStringArrayToUnorderedList } from '../../utils/generateUnorderedList';
+import { useQuery } from '@tanstack/react-query';
+interface DataType {
+    key: React.Key;
+    backendSystemType: BackendSystemStatus.backendSystemType;
+    name: string;
+    healthStatus: BackendSystemStatus.healthStatus;
+    endpoint: string | undefined;
+    details: string | undefined;
+}
+
+export const DisplayHealthCheckStatus = (): JSX.Element => {
+    const [nameFilters, setNameFilters] = useState<ColumnFilterItem[]>([]);
+    const [backendSystemTypeFilters, setBackendSystemTypeFilters] = useState<ColumnFilterItem[]>([]);
+    const [healthStatusFilters, setHealthStatusFilters] = useState<ColumnFilterItem[]>([]);
+    const [healthCheckError, setHealthCheckError] = useState<JSX.Element>(<></>);
+    const [backendSystemStatusList, setBackendSystemStatusList] = useState<DataType[]>([]);
+
+    const systemStatusQuery = useQuery({
+        queryKey: ['systemStatusQuery'],
```

Comment:
the queryKey must be different to what is there in `SystemStatusBar` component. 
Or if the query can be used for both SystemStatus and for HealthCheck, then we can use a custom query in a separate file in "systemStatus" package and use it from two places. This is more efficient.  

```typescript
export const useHealthCheckQuery = () => {
  return useQuery({
        queryKey: ['systemStatusQuery'],
        queryFn: () => AdminService.health(),
        staleTime: 60000, // one minute
    });
}
```